dev-tools/validation/test_check_authz_route_matrix.py

```
@@ -38,6 +38,204 @@ def setUpClass(cls):
+        document = self.validator.yaml.safe_load(MATRIX.read_text(encoding="utf-8"))
+        route = next(
+            route
+            for route in document["routes"]
+            if route.get("route") == "ToggleFeatureFlag"
+        )
+        route["accepted_token_profiles"] = ["control-ui", "unknown-profile"]
+        route["accepted_token_profile_audiences"] = {
+            "control-ui": "control-ui",
+            "unknown-profile": "unknown",
+        }
+        route["token_type"] = "control_plane_user"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "token_type"

(S105)